Entity/UserManager.php

```diff
@@ -146,4 +148,28 @@ public function getPager(array $criteria, $page, $limit = 10, array $sort = arra
 
         return $pager;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function loadUserByUsername($username): UserInterface
+    {
+        return $this->findUserByUsername($username);
```

Comment:
Please implement the `UserProviderInterface` strictly:

> This method must throw UsernameNotFoundException if the user is not found.

Reference: https://github.com/symfony/security-core/blob/2.3/User/UserProviderInterface.php#L39

Reply:
Ah, good spot.